internal/server/routes.go

```diff
@@ -107,6 +107,8 @@ func (s *Server) GenerateRoutes(promRegistry prometheus.Registerer) Routes {
 	post(a, noAuthn, "/api/signup", a.Signup)
 
 	post(a, noAuthn, "/api/login", a.Login)
+	post(a, noAuthn, "/api/password-reset-request", a.RequestPasswordReset)
+	post(a, noAuthn, "/api/password-reset", a.VerifiedPasswordReset)
```

Comment:
Is `/api/password-reset` a link from an email? Doesn't that have to be a `GET`? Or do email clients support `<form>` which allows us to make this a POST?

Reply:
no. /password-reset is the link from email. /api/password-reset is the api call that page makes.